\chapter{Combing human disease genetics and mouse model phenotypes towards drug repositioning: application on Parkinson's disease}\label{drug}

\section{Motivation}
Disease genetics information in
genome-wide association studies (GWAS) \cite{sanseau2012use} and
Online Mendelian Inheritance in Man (OMIM) \cite{wang2013rational} has great
potential to guide drug discovery.
In a recent drug repositioning study, Wang and Zhang
directly match the disease genes
in OMIM with the drug target genes to repositioning
existing drugs for new indications \cite{wang2013rational}.
Another approach proposed by Okada and colleagues
extends the disease associated genes in GWAS
with their functionally related genes based on
protein-protein interactions (PPIs), and matches
the extended gene set with the drug target genes
for drug repositioning \cite{okada2014genetics}.


On the other hand, studies on the underlying {\it in vivo}
biology of animal models are also useful in drug discovery.
The phenotypic descriptions for mouse genetic mutations
provide an in-depth understanding of gene functions,
thus allow us to gain new insights into human diseases
\cite{hoehndorf2011phenomenet}
and drug targets \cite{hoehndorf2014mouse}.
In a recent drug repositioning approach based on mouse phenotypes,
Hoehndorf and the team link human diseases to mouse phenotypes
through matching human and mouse phenotype ontologies.
They then compare mouse phenotype features for the disease and
all genes to predict disease-associated genes. After that,
they link the predicted disease-gene associations with
the drug-target data to suggest candidate drugs for a given disease \cite{hoehndorf2012linking}.

In this study, we developed a novel drug repositioning approach
leveraging both disease genetics and mouse model phenotypes.
Given a disease, we first identified disease-specific mouse phenotypes
using well-studied human disease genes. Then we searched
all the FDA-approved drugs for the candidates that share
similar mouse phenotype profiles with the disease.
We demonstrated the approach using Parkinson's disease (PD).
PD is the second most common neurodegenerative disorder and
currently lacks effective drug treatments \cite{olanow2009scientific}.
We used disease genes in OMIM to identify the
PD mouse phenotypes. To date, OMIM has included 15 high-penetrance
PD genes that are likely to cause the PD symptoms among
the mice carrying their mutations \cite{hamosh2005online}.
Even though these genes are mostly associated with familial PD,
clinical researches and association studies have shown that the
familial and sporadic forms of PD usually share the same molecular pathways
\cite{lesage2009parkinson,lesage2012role}. We ranked candidate drugs based on the
semantic similarities of mouse phenotype profiles between PD and the drugs.

We tested the ranking algorithm in prioritizing FDA-approved PD drugs and novel PD drugs. We compared our approach with the pure genetics-based approaches \cite{wang2013rational,okada2014genetics} and demonstrated that mouse model phenotypes are important for improving the performance of PD drug identification. We also compared with Hoehndorf's approach \cite{hoehndorf2012linking} and show that incorporating disease genetics using our novel approach achieves significantly better precision. We further examined the top-ranked drugs by comparing their gene expression profiles with that of PD.

\section{Data and methods}
Our hypothesis is that a drug has the potential to treat PD if the drug target genes
are associated with PD phenotypes. Gene-phenotype associations based on systematic
mouse gene knockouts provide rich information to link drugs and their new indications.
Fig. \ref{mphen} shows that our drug repositioning approach based on mouse phenotypes
contains two steps. In the first step, we searched for the mouse phenotypes associated with
PD using the well-studied disease genes. In the second step, we extracted a set of mouse
phenotype features for each candidate drug and systematically calculated the semantic similarities
(using mammalian phenotype ontology) of the phenotype profiles between PD and candidate drugs.
Using the mouse phenotype similarity between the drugs and disease,
we predicted how likely the drugs can be used to treat PD.
  \begin{figure}[h!]
  \begin{center}
\includegraphics[width=\textwidth]{Chap6_method.eps}
\end{center}
  \caption{Drug discovery approach for Parkinson’s disease combining human disease genetics and mouse mutation phenotypes. }\label{mphen}
  \end{figure}

  \subsection{Identify mouse model phenotypes for PD using disease genetics in OMIM }
We searched for mouse model phenotypes for PD using 15 genes associated with 20 subtypes of PD in OMIM.
The mutations of these genes highly increase the risk for PD and are likely to cause PD phenotypes.
All these human genes have homologies among mice. We downloaded the phenotype annotations for mouse genes
from Mouse Genome Informatics (MGI) \cite{eppig2015mouse}, and extracted 358 phenotypes that are linked to the 15 PD genes.
Different PD genes may share common phenotype annotations.
For example, 7 out of 15 PD genes point to the phenotype of neurodegeneration.
We weighted each phenotype with the number of its associated PD genes.
The weights intuitively represent the confidence that the phenotype is related with PD.

We ranked the PD-specific mouse model phenotypes by their weights,
and investigated the category of the top-ranked phenotypes.
The mammalian phenotype ontology classifies mouse phenotypes into 30 categories.
We first mapped each PD phenotype to its categories by tracing the $isa$
relationship in the mammalian phenotype ontology. The 358 phenotypes were mapped into 24 categories.
Then we calculated a score for each category by summing the weights of all the phenotypes in it.
We ranked the categories based on these scores and examined the top-ranked ones.

\subsection{Prioritize candidate PD drugs based on the similarities of mouse phenotype profiles between disease and drugs }
We collected a set of candidate drugs from DrugBank \cite{law2014drugbank}.
The drug-target database in DrugBank contains information for 1427 FDA-approved (for any indication) drugs.
We extracted 1197 drugs that target on human/mouse orthologous genes,
and included them into the candidate drug set. Then we combined the drug-target relationships
and phenotype annotations for the target genes to link each candidate drug to a set of
mouse model phenotypes through the drug target genes. We constructed a vector of mouse phenotypes
for each drug, and weighted each phenotype by the number of its associated target genes.

We calculated the semantic similarity between the vector of mouse phenotypes associated
with PD and each candidate drug to determine how likely the drug can be used to treat PD.
We first quantified the information content for each phenotype term $t$ as $-logp(t)$,
in which $p(t)$ represents the frequency among phenotype annotations to all the 7568 mouse genes.
In calculating the information content, if a gene is annotated by one phenotype term,
we assumed that it is also annotated by the ancestors of this term in the hierarchy of mammalian phenotype ontology.
Hence, a phenotype term has higher information content than its ancestors,
which lie on higher levels in the ontology. Then we defined the semantic distance $sim(t_1,t_2)$
between phenotype terms $t_1$ and $t_2$ as:
\begin{equation}
sim(t_1,t_2)=\max_{\alpha \in A(t_1,t_2)} -log p(a),
\end{equation}
where $A(t_1,t_2)$ is the set of common ancestors for $t_1$ and $t_2$. To calculate the distance
from the phenotype vector $p_1$ to $p_2$, we matched each phenotype term in $p_1$ to the
most similar term in $p_2$ and took the average:
\begin{equation}
sim(p_1 \to p_2)=avg(\sum\nolimits_{t_1 \in p_1}{\max_{t_2 \in p_2}sim(t_1,t_2)}).
\end{equation}
The matching similarity was weighted by the product of weights for phenotype term $t_1$ and $t_2$.
The similarity between $p_1$ and $p_2$ was defined as the average of semantic similarities in both directions:
\begin{equation}
sim(p_1,p_2 )=1/2 sim(p_1 \to p_2 )+1/2 sim(p_2 \to p_1 ).
\end{equation}
A similar calculation of semantic similarity between two vectors of ontology concepts was used before \cite{robinson2008human}.

\subsection{De novo evaluation in prioritizing FDA-approved PD drugs }
We investigated if our method can prioritize approved PD drugs. We ranked the 1197 candidate
drugs using the semantic similarities of the mouse phenotype profiles between the drugs and PD. Then we extracted approved PD drugs from FDA drug labels. Our drug ranking algorithm does not use any information of the approved PD drugs. In the de novo evaluation, we calculated the distribution of approved PD drugs among our ranks by plotting a 10-bin histogram. Specifically, we divided the ranks into 10 ranges, and counted the number of approved PD drugs within each range. In addition, we investigated the target genes for the top 10\% candidate drugs. We ranked these drug target genes by the number of drugs (ranked within top 10\%) that target on each gene. We also calculated the distribution of genes targeted by the FDA-approved drugs among all the drug target genes using histogram.

We demonstrated the importance of using mouse phenotypes to predict drugs for PD. Recent studies have shown that disease associated genes can guide the detection of existing drug therapies and promising candidate drugs \cite{sanseau2012use,wang2013rational}.  We compared our approach with two genetics-based drug discovery methods (Fig. \ref{mphen_comp}). The first method \cite{wang2013rational} directly matches the disease genes in OMIM with the drug target genes to repositioning existing drugs for new indications. The second method \cite{okada2014genetics} extends the disease genes with their functionally related genes based on protein-protein interactions (PPIs), and matched the extended gene set with the drug target genes for drug repositioning. We downloaded the PPIs from the STRING database \cite{snel2000string}, and used the experiment data source, which contains PPI databases such as HPRD, BIND, and GRID. We evaluated if the two methods have the ability to identify approved PD drugs without using mouse phenotypes, and compared the result with our approach.
  \begin{figure}[h!]
  \begin{center}
\includegraphics[width=.7\textwidth]{Chap6_compare.eps}
\end{center}
  \caption{We compared with genetics-based drug discovery methods, which directly match the disease genes and their interacting genes with the drug target genes. The comparison aims at demonstrating the importance of using mouse phenotypes.}\label{mphen_comp}
  \end{figure}

\subsection{Evaluation in ranking novel PD drugs and comparison with an existing drug repositioning approach}
We investigated if our approach has the ability to prioritize novel PD drugs. In our recent studies, we constructed large-scale drug-disease treatment knowledge bases from multiple data resources using techniques including natural language processing, text mining and data mining \cite{xu2013large,xu2013semi}. The databases included 9,216 drug-disease treatment pairs extracted from FDA drug labels, 34,306 pairs extracted from 22 million published biomedical literature abstracts, and 69,724 pairs extracted from 171,805 clinical trials. Based on these knowledge bases, we constructed two evaluation sets as the proxy of novel PD drugs: the first set consists of the drugs that have been tested for PD in clinical trials and the second set consists of PD drugs extracted from literature abstracts in Medline. We removed the FDA-approved PD drugs from both sets. We used histogram to investigate the distribution of drugs in each set among our rank. We also generated a precision-recall curve and calculated the mean average precision to evaluate the ranking of drugs in the union of the two sets.

We compared the performance of our approach with a recent drug discovery approach proposed by Hoehndorf \cite{hoehndorf2012linking}. In their approach, the human diseases were linked to mouse phenotypes through phenotype ontology comparison, and then associated with orthologous genes based on the gene-phenotype relationships in animal models. After that, they linked the predicted disease genes with the drug-target data to suggest candidate drugs for a given disease. We compared the histograms that represent the distributions of evaluation drugs as well as the precision-recall curves for the two methods.

\subsection{Test the top-ranked drugs using gene expression data analysis}
We further examined the top-ranked drugs by comparing their gene expression profiles in Gene Expression Omnibus (GEO) with that of PD. For the drugs, we extracted data sets that contain gene expression levels before and after adding the drugs to human or animal brain tissues. For PD, we downloaded the data sets that compared the PD patients and healthy controls. We used the GEO2R software \cite{barrett2013ncbi} to identify the significantly differential expressed genes (adjusted p value <0.05) for the disease and drugs, respectively. Then we investigated if common significant genes exist between PD and the drug, and if these common genes have opposite directions of regulation.

\section{Results}
\subsection{Our disease genetics-based phenotype prioritization algorithm identified PD-specific mouse model phenotypes }
 We ranked and classified the mouse model phenotypes detected using PD genes in OMIM. The top ranked phenotype categories are nervous system and behavior/neurological phenotypes as expected (Table \ref{mphenPD}). Examples of nervous system phenotypes with the highest weights include neurodegeneration and alpha-synuclein inclusion body, which characterize the pathology of PD. In addition, top-ranked behavior/neurological phenotypes, such as impaired coordination and abnormal gait, mostly include typical motor symptoms of PD. Interestingly, the rest top-ranked phenotype categories show that the pathology of PD is complex and involves not only the nervous system, but also immune system, homeostasis and other aspects.  \begin{table}[h!]
\caption{The top-ranked categories of mouse phenotypes extracted using PD genes in OMIM.}
  \label{mphenPD}
  \centering
      \begin{tabular}{ccc}
        \hline
         Rank  &Phenotype Category   &Example top-ranked phenotype\\ \hline
         1	&nervous system phenotype&	Neurodegeneration\\
2	&behavior/neurological phenotype	&impaired coordination\\
3	&immune system phenotype	&decreased double-positive t cell number\\
4	&homeostasis/metabolism phenotype	&decreased dopamine level\\
5	&hematopoietic system phenotype	&decreased hemoglobin content\\
\hline
\end{tabular}
\end{table}

\subsection{Our approach prioritized FDA-approved PD drugs}
We extracted 22 FDA-approved drugs for PD and 474 genes targeted by these drugs. The median rank of the 22 drugs is 125 (top 10\% among 1197 drugs). The histogram in fig. \ref{mphen_appdrug} shows that our approach prioritized 10 approved PD drugs within top 10\%. The table in fig. \ref{mphen_appdrug} shows the rank and percentile of the top 10 approved PD drugs. Among them, the most effective dopamine replacement agent, levodopa, was ranked within top 5\%. Fsig. \ref{mphen_appdrugtar} shows that the drugs prioritized by our approach frequently target on the drug target genes for approved PD drugs. In fig. \ref{mphen_appdrugtar}(a), nine in the top ten drug target genes (except GABRA1) are target genes for approved PD drugs. Fig. \ref{mphen_appdrugtar}(b) shows that half of the top 10\% genes have been targeted by approved PD drugs, while the other half are new drug targets and may lead to novel candidate PD drugs.
 \begin{figure}[h!]
  \begin{center}
\includegraphics[width=\textwidth]{Chap6_appdrugrank.eps}
\end{center}
  \caption{Our approach ranked the approved PD drugs in the top. A total of 10 among 22 approved PD drugs were ranked within top 10\% among all the 1197 drugs.}\label{mphen_appdrug}
  \end{figure}
   \begin{figure}[h!]
  \begin{center}
\includegraphics[width=\textwidth]{Chap6_appdrugtar.eps}
\end{center}
  \caption{The drug target genes that are most frequently targeted by our top 10\% drugs. (a) The top 10 drug target genes for our prioritized drugs. (b) The distribution of target genes for approved PD drugs among all the drug target genes.}\label{mphen_appdrugtar}
  \end{figure}

  Approved PD drugs and their target genes cannot be easily detected through matching disease genes and drug target genes. We compared the performance in identifying approved PD drugs with two genetics-based drug discovery methods. Using the first method, none of the 15 PD genes directly matches the target genes for approved PD drugs and we detected zero approved drug. Using the second method, we detected one approved PD drug, rasagiline, through its target gene BCL2, which interacts with the PD gene PARK2. Though the disease genes for PD and their interacting genes do not directly provide information on the drug target genes, our approach prioritized 10 out of 22 approved PD drugs by exploiting the gene-phenotype associations in mouse models.

  \subsection{Our approach outperformed an existing approach in prioritizing novel PD drugs }
  The top ranked drugs generated by our approach are enriched for the novel PD drugs in the two evaluation sets (fig. \ref{mphen_eva}). We extracted 81 drugs from clinical trials to construct the first set, and the candidate drugs in our approach contain 69 of them. Our approach ranked a total of 22 drugs in the top 10\%, and this number is 450\% higher than 4 drugs in the bottom 10\%. Most testing drugs (68\%) in the clinical trial set were ranked within top 30\%. The evaluation set based on Medline contains 102 drugs, and our candidate drugs included 85 among them. We ranked 26 within top 10\%, which is a 760\% increase comparing with 3 drugs in the bottom 10\%. In contrast, fig. \ref{control_eva} shows that the evaluation drugs spread out in different rank ranges when using the existing drug discovery approach based on mouse model phenotypes. Comparison between fig. \ref{mphen_eva} and \ref{control_eva} show that our approach performed better than Hoehndorf's approach in ranking novel PD drugs in the two evaluation sets.
     \begin{figure}[h!]
  \begin{center}
\includegraphics[width=\textwidth]{Chap6_evamphen.eps}
\end{center}
  \caption{The distribution of our ranks for two sets of novel PD drugs extracted from clinical trials and Medline texts. }\label{mphen_eva}
  \end{figure}
       \begin{figure}[h!]
  \begin{center}
\includegraphics[width=\textwidth]{Chap6_evacontrol.eps}
\end{center}
  \caption{The distribution of evaluation sets based on clinical trials and Medline texts among the ranks generated by the baseline approach based on mouse phenotypes. }\label{control_eva}
  \end{figure}

  The precision-recall curves in fig. \ref{mphen_pr} further shows that our performance is significantly better than the previous approach. The mean average precision for our approach is 0.24, which is significantly higher than 0.16 for the Hoehndorf's approach ($p<e^{-11}$). The result means that our approach achieved higher precision averagely at all recall levels, and mostly ranked the novel PD drugs higher than the previous approach.
         \begin{figure}[h!]
  \begin{center}
\includegraphics[width=.7\textwidth]{Chap6_prcurve.eps}
\end{center}
  \caption{Precision-recall curves in ranking the novel PD drugs for our approach and Hoehndorf’s approach based on PhenomeNet. }\label{mphen_pr}
  \end{figure}

  \subsection{Gene expression analysis suggests quetiapine as a potential PD drug}
  Among the top 10 candidate drugs, we found a set of gene expression samples available for quetiapine in GEO. We identified 61 significant genes for quetiapine from GEO series GSE4522933 and 1650 significant genes for PD from GSE839734. Table \ref{mphenGEO} lists the common significantly differential genes between PD and quetiapine, as well as the direction of regulation for each gene and the logarithm of fold change. Among these genes, MAOA regulates the metabolism of neurotransmitters such as dopamine and is closely associated with PD35. In addition, MAOA is not a drug target gene for quetiapine based on the drug-target data in DrugBank. The gene expression analysis suggests that quetiapine, one of the top ranked drugs, has the potential to treat PD.
   \begin{table}[h!]
\caption{Common significantly differential genes for PD and quetiapine as well as their directions of regulation and fold change.}
  \label{mphenGEO}
  \centering
      \begin{tabular}{ccccc}
        \hline
        \multirow{2}{*}{Gene}&\multicolumn{2}{l}{Quetiapine}&\multicolumn{2}{l}{PD}\\
                                                &regulation&Log(FC)&regulation&Log(FC)\\ \hline
        HSPB1	&Up	&1.5	&Down 	&-1.4\\
CHORDC1	&Up	&0.6	&Down	&-1.1\\
MAOA	&Down	&-0.6	&Up	&0.8\\
MRPL15	&Down	&-0.6	&Up	&0.8\\
SPEN	&Up	&0.3	&Down	&-0.5\\
EIF5	&Down	&-0.2	&Up	&0.4\\


\hline
\end{tabular}
\end{table}

\section{Discussion}
Currently, we used the disease genetics knowledge in OMIM as the seeds to detect PD mouse phenotypes. We have demonstrated in several recent works that disease genes predicted by analyzing human disease phenotype networks and genetic functional relationship networks also have the translational potential in drug discovery \cite{chen2014comparative,chen2014malaria,chen2014phenome}. In the future, we will develop approaches to integrate disease associated genes in OMIM, GWAS and prediction results from computational approaches in the drug repositioning approach. In addition, we will incorporate other information, including human disease phenotypes, disease similarities and drug similarities to further prioritize strong candidate drugs.


\section{Conclusions}
  In this study, we developed a novel drug repositioning approach to predict new drugs for Parkinson's disease using both disease genetics knowledge and mouse model phenotypes. Our approach can identify FDA-approved PD drugs and prioritize novel PD drugs. Comparison with pure genetics-based drug repositioning approaches shows the importance of mouse model phenotypes in identifying PD drugs. In addition, our approach outperformed a recently proposed mouse phenotype based drug discovery method through combining disease genetics with mouse model phenotypes using a novel computational approach. Further gene expression analysis on top-ranked candidate drugs suggested quetiapine as a potential PD therapy.
